ui/panels-plugin/src/plugins/gauge-chart/gauge-chart-model.ts

```diff
@@ -16,12 +16,13 @@ import { UnitOptions } from '@perses-dev/components';
 import { CalculationType, OptionsEditorProps } from '@perses-dev/plugin-system';
 
 export const DEFAULT_UNIT: UnitOptions = { kind: 'PercentDecimal' };
-
 export const DEFAULT_MAX_PERCENT = 100;
-
 export const DEFAULT_MAX_PERCENT_DECIMAL = 1;
 
-export type GaugeChartOptionsEditorProps = OptionsEditorProps<GaugeChartOptions>;
+export interface GaugeChart {
```

Comment:
I think you also need to export these from the `index.ts` file if you want them to be accessible for use outside perses internal dev, which I think was the goal with wanting to use these in converter code. Same comment goes for the other models.

Reply:
Thank you, I was wondering about that but didn't go all the way into figuring it out

Reply:
A good quick test if something is exported is to try out the quick and dirty `typedoc` command I added. If the value you want shows up in the docs it outputs, you know it is exported for public use.

Reply:
Looking at the code more closely because I hit a case I'd need this, should these types maybe have `Spec` at the end (e.g. `GaugeChartSpec`) and extend from the `Definition` interface here?  https://github.com/perses/perses/blob/main/ui/core/src/model/definitions.ts#L17-L20

Reply:
I was going to make the same comment that you can probably just use `Definition<GaugeChartOptions>`